testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/events/AbstractEventsTest.java

```diff
@@ -23,14 +23,15 @@
 import org.keycloak.testsuite.client.resources.TestingResource;
 import org.keycloak.testsuite.util.RealmBuilder;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  *
  * @author Stan Silvert ssilvert@redhat.com (C) 2016 Red Hat Inc.
  */
-public abstract class AbstractEventsTest extends AbstractKeycloakTest {
+public abstract class AbstractEventsTest extends AbstractKeycloakTest implements Serializable {
```

Comment:
Why is this change here?

Reply:
It was needed because I used the `realmId` property in the test. But I have changed it to get the realm from `session.getContext()` now. I removed these changes now..